include/RigidBody/SegmentCharacteristics.h

```diff
@@ -84,7 +84,7 @@ class BIORBD_API SegmentCharacteristics : public RigidBodyDynamics::Body
     /// \brief Returns the segment mass
     /// \return The segment mass
     ///
-    double mass() const;
+    const biorbd::utils::Scalar& mass() const;
```

Comment:
The return should not be a const ref (see previously)

Reply:
Done